package.json

```
@@ -4,10 +4,11 @@
   "description": "Weather app. Api https://weatherstack.com/",
   "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "bundle": "./node_modules/.bin/browserify src/index.js > dist/bundle.js"
+    "build:dev": "webpack --config ./webpack.config.dev.js",
+    "build:prod": "webpack --config ./webpack.config.prod.js",
+    "serve": "webpack serve --config ./webpack.config.dev.js"
```

```suggestion
    "start": "webpack serve --config ./webpack.config.dev.js"
```
Чаще всего скрипт для запуска дев сервера выглядит так, разработчику проще набрать команду `npm start` да и это можно назвать соглашением между разработчиками